new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful and popular editor that has many commands. Unlike most
+text editors where you use the keyboard to enter text and arrow keys to move you
+cursor, Vim uses the keyboard to enter commands to manipulate the text.
```

Comment:
Small note (in case this applies to other places): I think `, in Vim you use ...` makes more sense, as `Vim` isn't the one entering commands.